app/src/layouts/kanban/kanban.vue

```diff
@@ -185,6 +196,13 @@ function saveChanges() {
 	editDialogOpen.value = null;
 	editTitle.value = '';
 }
+
+// Used to find relevant information to display cards' formatted text with <render-display>
+const textFieldConfiguration = computed<Field | undefined>(() => {
+	/* Extra information is stored in fieldsInCollection,
+	but we need to find the config for the field used as the card's text field */
+	return props.fieldsInCollection.find((field) => field.field === props.layoutOptions.textField);
+});
```

Comment:
Code is self-explanatory, comments not necessary 👍 

```suggestion
const textFieldConfiguration = computed<Field | undefined>(() => {
	return props.fieldsInCollection.find((field) => field.field === props.layoutOptions.textField);
});
```